### ISSUE TITLE
Polybar dependencies problem in Ubuntu server 20.04

### ISSUE THREAD

**Issue by JavierReyes945** [regolith-linux/speed-ricer#12]
```
$ apt install polybar
The following packages have unmet dependencies:
 polybar : Depends: python-xcbgen but it is not installable
E: Unable to correct problems, you have held brocken packages

$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04 LTS
Release:        20.04
Codename:       focal
```

**Comment by ivi9901** [comment]
I've exactly the same problem, but within Pop!_OS 20.04 LTS.
I think it's because polybar 3.3 requires Python 2, whitch has come to an end of support, and newer versions of Ubuntu doesn't come with it. (I'm not totally sure, but for me it's the mos logical answer)
With the newer version (3.4.2) I don't have that problem, but it's a fuss to install it.

Would be great if this PPA upgrades his polybar version to the current one.

(Sorry for my terrible english)

**Comment by bashfulrobot** [comment]
I am just popping in here as I just ran into the same issue.

**Comment by kgilmer** [comment]
I spent some time trying to compile 3.4.3 with pbuilder but there is some cmake issue with [Boost and pthreads](https://github.com/polybar/polybar/issues/259).  Or something.  I'm not going to push changes since I couldn't get it working but if anyone has advice or something occurs to me in the shower then I'll take another look.  Unraveling the mystery of passing CMAKE flags properly through all of the build layers from Debian to the actual build seems beyond me at the moment.

```
Run Build Command:"/usr/bin/make" "cmTC_13124/fast"
make[1]: Entering directory '/build/polybar-3.4.3-2/obj-x86_64-linux-gnu/CMakeFiles/CMakeTmp'
/usr/bin/make -f CMakeFiles/cmTC_13124.dir/build.make CMakeFiles/cmTC_13124.dir/build
make[2]: Entering directory '/build/polybar-3.4.3-2/obj-x86_64-linux-gnu/CMakeFiles/CMakeTmp'
Building CXX object CMakeFiles/cmTC_13124.dir/CheckSymbolExists.cxx.o
/usr/bin/c++    -g -O2 -fdebug-prefix-map=/build/polybar-3.4.3-2=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -Wall -Wextra -Wpedantic    -o CMakeFiles/cmTC_13124.dir/CheckSymbolExists.cxx.o -c /build/polybar-3.4.3-2/obj-x86_64-linux-gnu/CMakeFiles/CMakeTmp/CheckSymbolExists.cxx
Linking CXX executable cmTC_13124
/usr/bin/cmake -E cmake_link_script CMakeFiles/cmTC_13124.dir/link.txt --verbose=1
/usr/bin/c++  -g -O2 -fdebug-prefix-map=/build/polybar-3.4.3-2=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -Wall -Wextra -Wpedantic   -Wl,-Bsymbolic-functions -Wl,-z,relro  -rdynamic CMakeFiles/cmTC_13124.dir/CheckSymbolExists.cxx.o  -o cmTC_13124 
/usr/bin/ld: CMakeFiles/cmTC_13124.dir/CheckSymbolExists.cxx.o: in function `main':
./obj-x86_64-linux-gnu/CMakeFiles/CMakeTmp/./obj-x86_64-linux-gnu/CMakeFiles/CMakeTmp/CheckSymbolExists.cxx:8: undefined reference to `pthread_create'
collect2: error: ld returned 1 exit status
make[2]: *** [CMakeFiles/cmTC_13124.dir/build.make:87: cmTC_13124] Error 1
make[2]: Leaving directory '/build/polybar-3.4.3-2/obj-x86_64-linux-gnu/CMakeFiles/CMakeTmp'
make[1]: *** [Makefile:121: cmTC_13124/fast] Error 2
make[1]: Leaving directory '/build/polybar-3.4.3-2/obj-x86_64-linux-gnu/CMakeFiles/CMakeTmp'

File /build/polybar-3.4.3-2/obj-x86_64-linux-gnu/CMakeFiles/CMakeTmp/CheckSymbolExists.cxx:
/* */
#include <pthread.h>

int main(int argc, char** argv)
{
  (void)argv;
#ifndef pthread_create
  return ((int*)(&pthread_create))[argc];
#else
  (void)argc;
  return 0;
#endif
}

dh_auto_configure: cd obj-x86_64-linux-gnu && cmake -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=None -DCMAKE_INSTALL_SYSCONFDIR=/etc -DCMAKE_INSTALL_LOCALSTATEDIR=/var -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON "-GUnix Makefiles" -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_INSTALL_LIBDIR=lib/x86_64-linux-gnu .. returned exit code 1
make: *** [debian/rules:5: build] Error 255
```

**Comment by rehashedsalt** [comment]
@kgilmer Not sure what the issue you were facing was, but I was able to build it just fine with almost no tweaks. It required a switch to the Python 3 version of `xcbgen` like we discussed over email, but a vanilla Focal pbuilder (`pbuilder --distribution focal --architecture amd64`) packed it together just fine for me.

Using upstream sources (polybar/polybar), tag 3.4.3. I'm gonna give it a couple tests with my configs and open a PR shortly.
